ngraph_bridge/ngraph_mark_for_clustering.cc

```diff
@@ -136,6 +138,30 @@ static ConfirmationFunction SimpleConfirmationFunction() {
   return cf;
 };
 
+// Check if op is supported by backend using is_supported API
+static Status IsSupportedByBackend(
+    const Node* node,
+    std::map<std::string, std::vector<shared_ptr<ng::Node>>>& TFtoNgraphOpMap,
+    const string& backend_name, bool& is_supported) {
```

Comment:
as discussed we could pass in an actual backend, rather than a string name. That will help in writing tests